blocks/text.js

```diff
@@ -689,3 +689,90 @@ Blockly.Blocks['text_prompt'] = {
   mutationToDom: Blockly.Blocks['text_prompt_ext'].mutationToDom,
   domToMutation: Blockly.Blocks['text_prompt_ext'].domToMutation
 };
+
+Blockly.Blocks['text_count'] = {
+  /**
+   * Block for counting how many times one string appears within another string.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      'message0': Blockly.Msg.TEXT_COUNT_MESSAGE0,
+      'args0': [
+        {
+          'type': 'input_value',
```

Comment:
Oh okay. So even though it's not JSON string, it's a JavaScript object literal that's being passed to an API that's expecting a decoded JSON string, you want the object literal to be written as if it was a JSON string? That seems a bit strange to me, but sure.

Is this process of porting to Android and iOS automatic? I didn't see any warnings about this in the documentation for `jsonInit`.